sensors/video/sensorhub-driver-rtpcam/src/main/java/org/sensorhub/impl/sensor/rtpcam/RTSPClient.java

```diff
@@ -246,7 +242,7 @@ private void addDigestAuth(String method, String digestUri) throws IOException
     {
         try
         {
-            MessageDigest md = MessageDigest.getInstance("MD5");
+            MessageDigest md = MessageDigest.getInstance("SHA-256");
```

Comment:
It's not sufficient to switch the digest algorithm. You must also add the `algorithm` token to the `Authorization` header.

Reply:
I pushed the "fix" for this, but can't test it with a real camera until I manage to make a visit to the office in Huntsville to check one out.